pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.3
+lockfileVersion: 5.4
```

Comment:
weirdly this was specifying a lockfile format of **5.3** even though **5.4** is the one used within pnpm7